hw1/01-fizzbuzz.js

```diff
@@ -10,7 +10,21 @@ Use console.log() to write the proper output to the command line.
 **/
 
 const fizzbuzz = () => {
-  // Add your code here
+  for (let i = 0; i < 100; i++) {
+    curr = (i + 1)
+    if (curr % 3 === 0 && curr % 5 === 0) {
+      console.log("fizzbuzz")
+    }
+    else if (curr % 3 === 0) {
+      console.log("fizz")
+    }
+    else if (curr % 5 === 0) {
+      console.log("buzz")
+    }
+    else {
+      console.log(curr)
```

Comment:
instead of multiple console.log use it at the end. You can define a variable, and for each case you can assign it.